packages/slate-react/src/components/editor.js

```diff
@@ -260,12 +265,11 @@ class Editor extends React.Component {
 
   onChange = change => {
     debug('onChange', { change })
-
-    this.stack.run('onChange', change, this)
-    const { value } = change
+    if (this.value === change.value) return
+    const value = this.processValueOnChange(change.value, this.stack)
     const { onChange } = this.props
-    if (value == this.value) return
-    onChange(change)
+    if (value === this.value) return
```

Comment:
isnt `this.value` just a getter that resolves to the same thing as like `269`?

Reply:
Yes.